modules/amf-registration/amf-registration-rest/amf-registration-rest-impl/src/main/java/com/liferay/amf/rest/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -13,4 +16,32 @@
 	scope = ServiceScope.PROTOTYPE, service = AccountResource.class
 )
 public class AccountResourceImpl extends BaseAccountResourceImpl {
+
+	@Override
+	public Account createAccount(Account account) throws Exception {
+		
+		_accountLocalService.createAccount(
+			account.getFirstName(),
+			account.getLastName(),
+			account.getEmailAddress(),
+			account.getAccountName(),
+			account.getGenre(),
+			account.getBirthday(),
+			account.getPassword(),
+			account.getHomePhone(),
+			account.getMobilePhone(),
+			account.getAddress1(),
+			account.getAddress2(),
+			account.getCity(),
+			account.getState(),
+			account.getSecurityQuestion(),
+			account.getSecurityAnswer()
+		);
+
+		return account;
```

Comment:
use the `mapper` class to convert the `Account` (model) to `Account` (DTO) and return this last one. so we make sure everything was assigned and returned properly (my opinion).